wct.conf.json

```diff
@@ -1,8 +1,5 @@
 {
   "plugins": {
-    "local": {
-      "browsers": ["chrome"]
```

Comment:
Firefox still does not seem to be working on Travis. Not sure if we want to remove this config...